README.md

```diff
@@ -1 +1,35 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+In this project, we set up a repository for our portfolio website as software developers. For now we just developed the mobile view based on the requirements given, and also to practice flexbox, background image and more on linters.
+
+Built With
+
+HTML
+CSS
+HTML & CSS Linter
+
+Getting Started
+
+Please get your browser update. To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal: git clone in your terminal cd hello-microverse start index.html Authors
+
+👤 Author
+
+GitHub: [github](https://github.com/VSeray)
+Twitter: [twitter](https://twitter.com/home)
+LinkedIn: [Linkdin](https://www.linkedin.com/in/vana-seraydarian-936687191/?lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3BNyso4dw6Tz6UBL%2Fqkjvtvw%3D%3D)
+
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page. Show your support
+
+Give a ⭐️ if you like this project! Acknowledgments
+
+Entire Microverse team
+My learning and coding partners
+
+📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] Please kindly add a link to MIT license to make your `readme` look professional 👍🏻